### DISCUSSION TITLE
Is self-hosted or gratis hosted Weblate appropriate for open fan translation projects of non-software media?

### DISCUSSION THREAD

**Discussion by kevinlul** [WeblateOrg/weblate#9616] [General] [answered]
I happened upon Weblate and it looks to be a very useful application in general. Browsing https://hosted.weblate.org/, I'm finding that most of the projects are for managing translations of software, e.g. UI strings. I'm looking for a tool that could aid in community translations of Yu-Gi-Oh cards and interoperate with GitHub. These cards are revealed in one region first, typically Japan, and take months to be imported to other regions by Konami. In the interim, community-driven translations are the best way for international players to understand upcoming product. The strings to be translated are the names of the cards (typically <20 characters, but could be about three times longer) and the card text (entire paragraphs). The source language would most often (but not always) be Japanese, translating to Korean and, in the future, other European languages. There is no need for templating or plurals that are necessary for software UI strings. Is self-hosted or gratis hosted Weblate appropriate for this type of project? Thank you for your consideration.

**Comment by nijel** [comment, marked as answer]
The gratis hosting is not limited by content type, but by a license:

> Translated content has to be released under a license [approved by OSI](https://opensource.org/licenses) or [recognized as libre by FSF](https://www.gnu.org/licenses/license-list.html).

**Reply by kevinlul** [reply]
Hi, we'd like to proceed with this option. Should we set up the repository fully first and then contact through the website?

**Reply by nijel** [reply]
Yes, you can start here: https://hosted.weblate.org/hosting/
